src/less/chat.less

```diff
@@ -10,7 +10,14 @@
   font-size: 13px;
   position: relative;
 }
-
+#chat-log .message.whisper {
+  background: #e8e8ef;
+  border: 5px solid #545469;
+}
+#chat-log .message.blind {
+  background: #ffebff;
+  border: 5px solid #6b476b;
```

Comment:
Still not liking the thicker border here. But, as above, the color is fine, if we can't find another color to take its place.

Reply:
https://colour-palette-generator.azurewebsites.net/?r=216&g=127&b=214